docs/typescript.md

```diff
@@ -11,3 +11,6 @@ The fact that different parts of this project are built for different target arc
 The configuration of the `src/` files is such that code to be executed…
 * **in the client** can make use of all files but the ones in`src/server` and use all types but the ones defined in `src/types/server`
 * **on the server** can make use of all files but the ones in`src/client` and use all types but the ones defined in `src/types/client`
+
```

Comment:
Having this here seems a bit odd, since it doesn't have anything to do with TypeScript. It could get its own file, `docs/wikibase-integration.md` perhaps?